[#1422] Add section for kubectl (#1445)
airyhq/airy: docs/docs/getting-started/installation/minikube.md

If you want to customize your `Airy Core` instance please see our [Configuration
Section](configuration.md).

> After the installation, you can additionally interact with the components of `Airy Core` with the [kubectl](https://kubernetes.io/docs/tasks/tools/) command line utility, for example:

Reviewer: ```suggestion
After the installation, you can also interact with the components of `Airy Core` with the [kubectl](https://kubernetes.io/docs/tasks/tools/) command line utility:
```

If you want to customize your `Airy Core` instance please see our [Configuration
Section](configuration.md).

After the installation, you can also interact with the components of `Airy Core` with the [kubectl](https://kubernetes.io/docs/tasks/tools/) command line utility.

> The kubeconfig for the installed `Airy Core` is added to the default kubeconfig file `~/.kube/config`, under the context `airy-core`.

Reviewer: ```suggestion
`airy create` adds the kubeconfig of your Airy Core instance to the default kubeconfig file `~/.kube/config`, under the context `airy-core`.
```
